### ISSUE TITLE
Captcha *still* not working in Firefox

### ISSUE THREAD

**Issue by emp-irium** [hCaptcha/hcaptcha-wordpress-plugin#353]
Hello 👋,

**This is in reference to an existing issue** that has been marked as completed and can be found here: 
[https://github.com/hCaptcha/hcaptcha-wordpress-plugin/issues/342](https://github.com/hCaptcha/hcaptcha-wordpress-plugin/issues/342)

We replied at the very end of it, but as the case has been closed we assume that no one will pay attention to it, and thus we are creating a new issue to make it visible again - we hope this isn't causing any problems 👍

We waited for the update and to check out the situation for quite some time - even waited for another update - but the issue persists. We are unable to verify that hcaptcha works while using Firefox. Even the [login link](https://test.kagg.eu/wp-login.php) you provided results in this screen:

![screenshot-issue](https://github.com/user-attachments/assets/48040e84-86a0-4efa-92d0-8bd4d2adec1e)

In other words, the hcaptcha frame cannot be interacted with and thus makes it impossible to verify any login or comment reply through forms.

What else can be done?
